src/shared_utilities/utils.jl

```diff
@@ -686,6 +680,19 @@ function landsea_mask(
         Interpolations.Flat(),
     ),
 )
+    # Assuming 1 degree is 100 km = 1e5 m, we can estimate the resolution in
+    # degrees by taking the nodal_distance_meters and diving by 1e5
+    nodal_distance_in_meters =
+        ClimaCore.Spaces.node_horizontal_length_scale(surface_space)
+    one_degree_in_meters = 100_000
+
+    nodal_distance_in_degrees = nodal_distance_in_meters / one_degree_in_meters
+    nodal_distance_in_arcsec = nodal_distance_in_degrees * 3600
+
+    # Pick the landsea mask at 60 arcseconds if the nodal distance is more than
+    # 120", otherwise pick the higher resolution
+    resolution = nodal_distance_in_arcsec > 120 ? "60arcs" : "30arcs"
```

Comment:
I'm not really a fan of the hard-coded things here, but if they really will never change, that's fine with me.